lib/patch-series.ts

```diff
@@ -106,7 +113,7 @@ export class PatchSeries {
                 }
             });
 
-            rangeDiff = await git(["range-diff", "--creation-factor=95", "--no-color", range]);
+            rangeDiff = { previousRange: range, baseCommit, headCommit };
```

Comment:
I just noticed that this is a `...` range.. Maybe we would want to adjust this to define `previousRange` to `<current-branch>..<latest-tag>` and `currentRange` to `<latest-tag>..<current-branch>`?

Reply:
I am glad you brought this up.  I was meaning to ask.  Do I need to convert this range to hashes?  `gitShortHash` could just return the value if it does not look like a hash (or do not call `gitShortHash` if it does not look like a hash.  Certainly, changing to specify `currentRange` will make other code simpler.  Will other reviewers be able to get the tags if they are shown in the 'no range-diff' message?